pennylane/drawer/tape_mpl.py

```diff
@@ -119,13 +122,19 @@ def _tape_mpl(tape, wire_order=None, show_all_wires=False, decimals=None, **kwar
         for op in layer_ops:
             specialfunc = special_cases.get(op.__class__, None)
             if specialfunc is not None:
-                mapped_wires = [wire_map[w] for w in op.wires]
+                mapped_wires = (
+                    [wire_map[w] for w in op.wires] if len(op.wires) != 0 else wire_map.values()
```

Comment:
It might be nice to add a small comment to say that if op.wires==0 we assume that it acts on all wires.